Formalisation(shacl)/Core/PiecesShape/Dataset.ttl

```diff
@@ -1,177 +1,391 @@
+    <http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#status>, <http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#temporal-coverage>,
+    <http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#temporal-resolution>, <http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#theme> .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#modification-date> sh:datatype
+    xsd:dateTime;
+  sh:description "The most recent date on which the Dataset was changed or modified."@en;
+  sh:maxCount 1;
+  sh:name "modification date"@en;
+  sh:nodeKind sh:Literal;
+  sh:path dct:modified .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#number-of-records> sh:datatype
+    xsd:nonNegativeInteger;
+  sh:description "Size of the dataset in terms of the number of records"@en;
+  sh:maxCount 1;
+  sh:minCount 1;
+  sh:name "number of records"@en;
+  sh:nodeKind sh:Literal;
+  sh:path healthdcatap:numberOfRecords .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#number-of-unique-infividuals>
+  sh:datatype xsd:nonNegativeInteger;
+  sh:description "Number of records for unique individuals."@en;
+  sh:maxCount 1;
+  sh:name "number of unique infividuals"@en;
+  sh:nodeKind sh:Literal;
+  sh:path healthdcatap:numberOfUniqueIndividuals .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#other-identifier> sh:description
+    "A secondary identifier of the Dataset, such as MAST/ADS17, DataCite18, DOI19, EZID20 or W3ID21"@en;
+  sh:name "other identifier"@en;
+  sh:nodeKind sh:IRI;
+  sh:path adms:identifier .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#personal-data> sh:description
+    "Key elements that represent an individual in the dataset."@en;
+  sh:name "personal data"@en;
+  sh:nodeKind sh:IRI;
+  sh:path dqv:hasPersonalData .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#population-coverage> sh:datatype
+    xsd:string;
+  sh:description "A definition of the population within the dataset"@en;
+  sh:name "population coverage"@en;
+  sh:nodeKind sh:Literal;
+  sh:path healthdcatap:populationCoverage .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#publisher> sh:description
+    "An entity (organisation) responsible for making the Dataset available."@en;
+  sh:maxCount 1;
+  sh:minCount 1;
+  sh:name "publisher"@en;
+  sh:node foaf:Agent;
+  sh:path dct:publisher .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#publisher-note> sh:datatype
+    xsd:string;
+  sh:description "A description of the publisher activities"@en;
+  sh:maxCount 1;
+  sh:name "publisher note"@en;
+  sh:nodeKind sh:Literal;
+  sh:path healthdcatap:publishernote .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#publisher-type> sh:description
+    "A type of organisation that makes the Dataset available"@en;
+  sh:maxCount 1;
+  sh:name "publisher type"@en;
+  sh:nodeKind sh:IRI;
+  sh:path healthdcatap:publishertype .
```

Comment:
See the issue raised in https://github.com/Health-RI/health-ri-metadata/issues/166.

```suggestion
  sh:path healthdcatap:publisherType .
```